portfolio/src/main/webapp/style.css

```diff
@@ -49,4 +49,57 @@
   padding: 15px;
   width: 60%;
   margin: 10px;
+}
+
+#photoscontent {
+  margin: auto;
+  width: 90%;
+  text-align: center;
+}
+#photoscontent #photosblocks {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 4px;
+}
+#photoscontent .photoblock {
+  position: relative;
+  flex: 45%;
+  max-width: 45%;
+  margin: 20px;
+}
+#photoscontent .photoblock h3 {
+  position: absolute;
+  top: 4%;
+  left: 5%;
+  font-size: 1.5vw;
+}
+#photoscontent .photoblock h4 {
+  position: absolute;
+  top: 9%;
+  left: 5%;
+  font-size: 1.2vw;
```

Comment:
btw, just to clarify -- for this and the font-size as well above, i see you're using "vw" for viewport size instead of `em` or `px`.

since you're also using @media(screen....) in the css code below, i assume you're doing this because you want some form of fluid changes in the typography as the screen size changes without setting those breakpoints of sizes yourself?

Reply:
yup! is using "vw" okay?

Reply:
Ya using viewport sized is fine/good when you want some fluid typography + doing some layouts more easily.

Just the caveat that this only becomes possibly problematic with small screen-sizes (ie with 'vw' small screen width), since the text gets scaled according to the viewport width. Curious to how the text/font looks with the small width constraint and whether it is still readable.